bcipy/helpers/load.py

```diff
@@ -38,6 +41,44 @@ def copy_parameters(path: str = DEFAULT_PARAMETERS_PATH,
     return path
 
 
+def load_experiments(path: str=f'{DEFAULT_EXPERIMENT_PATH}{EXPERIMENT_FILENAME}') -> dict:
+    """Load Experiments.
+
+    PARAMETERS
+    ----------
+    :param: path: string path to the experiments file.
+
+    Returns
+    -------
+        A dictionary of experiments, with the following format:
+            { name: { fields : {name: '', required: bool}, summary: '' } }
+    
+    """
+    with open(path, 'r') as json_file:
+        return json.load(json_file)
+
```

Comment:
These functions don't guarantee that the data will have the documented fields. The structure is assumed in a number of places but not enforced. It may be worth doing a check on load or converting the JSON to a namedtuple or object.

Reply:
I have a validate function in the next PR for Experiment Fields! I had to break this up after it started getting bulky.